Mathlib/LinearAlgebra/ExteriorAlgebra/Basic.lean

```diff
@@ -66,6 +71,21 @@ def ι : M →ₗ[R] ExteriorAlgebra R M :=
   CliffordAlgebra.ι _
 #align exterior_algebra.ι ExteriorAlgebra.ι
 
+section exteriorPower
+
+variable (n : ℕ) (N : Type u2) [AddCommGroup N] [Module R N]
+-- New variables `n` and `N`, to get the correct order of variables in the notation.
+
+/-- Definition of the `n`th exterior power of a `R`-module `N`. We introduce the notation
+`Λ[R]^n N` for `exteriorPower R n N`. -/
+@[reducible]
+def exteriorPower := LinearMap.range (ι R : N →ₗ[R] ExteriorAlgebra R N) ^ n
```

Comment:
Just to be explicit here
```suggestion
abbrev exteriorPower : Submodule R (ExteriorAlgebra R M) :=
  LinearMap.range (ι R : N →ₗ[R] ExteriorAlgebra R N) ^ n
```
(and `@[reducible] def` is basically the same as `abbrev`)

Reply:
(I applied this change)